app/mailers/user_mailer.rb

```diff
@@ -25,16 +25,25 @@ def activate_account_email
   end
 
   def invitation_email
+    @email = params[:email]
     @name = params[:name]
     @signup_url = params[:signup_url]
     @email = params[:email]
 
     mail(to: @email, subject: t('email.invitation.invitation_to_join'))
   end
 
+  private
+
+  def preset
```

Comment:
`preset` was introduced to enable us to access `@provider` and `@base_url` outside of `branding` context in case we need it,
Looking at this now, I think it's not that necessary at the moment and can be placed inside of `branding` unless that we need it.
What you guys think?